src/main/java/TaskList.java

```diff
@@ -0,0 +1,43 @@
+public class TaskList {
+    private Task[] tasks = new Task[100];
+    private int taskCount = 0;
+
+    public int getTaskCount(){
+        return taskCount;
+    }
+
+
+    public void addTask(Task task){
+        if(taskCount == 100){
+            System.out.println("Duke can store 100 task at most.");
+        } else{
+            tasks[taskCount] = task;
+            taskCount++;
+        }
+    }
+
+    public void setTaskDone(int taskIndex){
+        tasks[taskIndex].setTaskDone(true);
+    }
+
+    public Task getTask(int taskIndex){
+        return tasks[taskIndex];
+    }
+
+    public void printTaskList(){
+        for(int i=0; i<taskCount; i++){
+            System.out.println("\t" + (i+1) + "." + tasks[i].getTaskInfoFormat());
+        }
+    }
+
+    public int getTasIndex(String taskName){
+        for(int i=0; i<taskCount; i++){
+            if(taskName.equals(tasks[i].getTaskName())){
+                return i;
+            }
+        }
+        return -1;
```

Comment:
This -1 seems slightly out of place, maybe you can replace it with a static final variable or a constant? I could not tell the reason for it.